### PR TITLE
feat(wiki): polish landing page and entry points

### PR DESCRIPTION
**Pull request by DannyS03**
## Context

- Simplifies content on the landing page.
- Reduces some of the JS bloat from previously used functions for the landing page.
- Focusses on "current state" and updates the cards accordingly.
- Updates announcement bar to callout Polygon 2.0.
- Various improvements to navbar, in attempt to provide relevant sections and to better utilize the space.
- Removes not so necessary "improve wiki" section that lives at the bottom of the landing page.
- Changes "Protocols" to "Polygon Stack".
- Governance is now referenced on the navbar.
- Other general improvements.

### CODE REVIEW

File: src/data/features.js [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
+        linkUrl: "docs/operate/technical-requirements",
         imageUrl: "img/logo-round-dark.png",
-        description: "Become a Polygon validator by operating a node and participating in consensus."
+        description: "Contribute to decentralized computing by running your own PoS node."
     },
     {
-        title: "Participate in Governance",
-        linkUrl: "/docs/maintain/govern/pips/",
+        title: "Run a zkEVM node",
+        linkUrl: "docs/operate/technical-requirements",
```
Comment (ashucoder9) [inline src/data/features.js:28]:
```suggestion
        linkUrl: "https://zkevm.polygon.technology/docs/setup-local-node",
```

File: docusaurus.config.js [hunk 2]
```diff
@@ -550,238 +465,38 @@ module.exports = {
...
-                    href: '/docs/polygonid/overview',
-                    label: 'Polygon ID',
-                    target: '_self',
-                    rel: null,
-                  },
-                ],
-        }, */
-        /*
+          to: '/docs/maintain/govern/governance-pos',
+          label: 'Participate in Governance',
```
Comment (ashucoder9) [inline docusaurus.config.js:470]:
```suggestion
          label: 'Governance',
```

File: src/data/features.js [hunk 3]
```diff
@@ -1,39 +1,39 @@
...
         imageUrl: "img/logo-round-dark.png",
-        description: "Learn about governance and shape the future of Polygon through the PIP framework."
+        description: "Contribute to decentralized computing by running your own ZK node."
     },
     {
-        title: "Become a Contributor",
-        linkUrl: "docs/contribute/orientation/",
+        title: "Run a Miden VM",
+        linkUrl: "docs/operate/technical-requirements",
```
Comment (ashucoder9) [inline src/data/features.js:34]:
```suggestion
        linkUrl: "docs/miden/user_docs/usage",
```

File: src/pages/index.js [hunk 4]
```diff
@@ -222,10 +84,9 @@ function Home() {
                 <h1 className="mt-0"><a href="https://polygon.technology" class="landing-page-link">Polygon</a> Wiki</h1>
                 <h3 className="mt-0"> The official documentation for <a href="https://polygon.technology" class="landing-page-link">0xPolygon</a></h3>
                 <p className="lead">The <b>Polygon Wiki</b> is the source of truth for Polygon, providing comprehensive documentation, community resources, and guides for enthusiasts and developers interested in learning about or building on Polygon.</p>
-                <a href="docs/home/new-to-polygon/" style={{ color: '#ffffff' }}>
-                  <button className="btn btn-custom">Get started with Polygon</button>
+                <a href="https://university.polygon.technology/" style={{ color: '#ffffff' }}>
```
Comment (ashucoder9) [inline src/pages/index.js:87]: University link surely abstracts away some complexity but it's not live up to the expectations of users. imo, we can remove this tab until Univ goes live or we build proper learning-focused content.

**Review by ashucoder9 — APPROVED** [review]

**Comment by ashucoder9** [comment]
Also, I think Infra provider is an important part of the Polygon ecosystem. We should find a way to showcase it.

**Comment by DannyS03** [comment]
> @DannyS03 , I think the Infra provider is an important part of the Polygon ecosystem. We should find a way to showcase it. Rest all of the changes look good to me.

Right, I agree. However, it's detracting from the navbar's overall usefulness. We've listed the providers in our builder materials, which also includes a key reference to a content explainer, a detail not covered in the dropdown menu.

**Comment by DannyS03** [comment]
Thanks for the review @ashucoder9

File: src/pages/index.js (hunk 4 above)
Comment (DannyS03) [reply src/pages/index.js:87]: We should only use this in the meantime.

**Pull request merged**